### PR TITLE
Redesign of Egardia component and platform.

### PR DESCRIPTION
**Pull request by jeroenterheerdt**
**Description:**
Updating docs related to the redesign of the Egardia component and platform.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#11994

## Checklist:

- [X] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by jeroenterheerdt** [comment]
Travis build failed because of time-out.

**Comment by jeroenterheerdt** [comment]
@MartinHjelmare here are the docs for HASS PR 11994. Any chance we can get them reviewed and merged so they are released in sync with the PR?

File: source/_components/binary_sensor.egardia.markdown [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+layout: page
+title: "Egardia Binary Sensor"
+description: "Instructions how to integrate Egardia / Woonveilig binary sensors into Home Assistant."
+date: 2018-03-02 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: egardia.png
+ha_release: 0.47
```
Comment (MartinHjelmare) [inline source/_components/binary_sensor.egardia.markdown]: 0.65

File: source/_components/binary_sensor.egardia.markdown [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ha_category: Binary Sensor
+ha_iot_class: "Local Push"
```
Comment (MartinHjelmare) [inline source/_components/binary_sensor.egardia.markdown]: This platform is polling.

File: source/_components/egardia.markdown [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+layout: page
+title: "Egardia"
+description: "Instructions how to setup Egardia / Woonveilig within Home Assistant."
+date: 2018-03-02 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Hub
+ha_release: 0.47
```
Comment (MartinHjelmare) [inline source/_components/egardia.markdown]: 0.65

File: source/_components/egardia.markdown [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+- **version** (*Optional*): The version of the Egardia system. `GATE-01`, `GATE-02` and `GATE-03` are currently supported. Defaults to `GATE-01`.
+- **port** (*Optional*): The port of the alarm panel. Defaults to 80.
+- **report_server_enabled** (*Optional*): Enable reporting by server. Defaults to `False`.
+- **report_server_port** (*Optional*): Port of the Egardia server. Defaults to 52010.
+- **report_server_codes** list (*Optional*): List of codes for the different states.
+
+Note that this basic configuration will only enable you to read the armed/armed away/disarmed status of your alarm and will **not** update the status if the alarm is triggered. This is because of how Egardia built their system. The alarm triggers normally go through their servers.
+You can change this, however, using the following procedure. This is a more advanced configuration.
+
+1. Log in into your alarm system's control panel. You will need to access http://[IP of your control panel]. You know this already since you need it in the basic configuration from above. Log in to the control panel with your Egardia/Woonveilig username and password.
```
Comment (MartinHjelmare) [inline source/_components/egardia.markdown]: "Log in to your alarm system's..."

File: source/_components/egardia.markdown [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+3. On your Home Assistant machine run `$ sudo python3 egardiaserver.py`. Refer to the [python-egardia repository](https://github.com/jeroenterheerdt/python-egardia) for detailed documentation on parameters. This will receive status codes from your alarm control panel and display them. You will need the codes to include in your configuration.yaml. Make sure to change the status of your alarm to all states (disarm, arm, home) by all means possible (all users, remotes, web login, app) as well as trigger the alarm in all ways possible to get 100% coverage. **Before triggering the alarm it might be good to disable the siren temporarily (can be done in Panel Settings).**
```
Comment (MartinHjelmare) [inline source/_components/egardia.markdown]: This step needs updating since we now start the server from home assistant.

Comment (jeroenterheerdt) [reply source/_components/egardia.markdown]: I do not agree - the `egardiaserver.py` script serves in this step to capture alarm codes. The user is asked here to run the script and capture the codes so they can be put into the config.

File: source/_components/binary_sensor.egardia.markdown (hunk 1 above)
Comment (jeroenterheerdt) [reply source/_components/binary_sensor.egardia.markdown]: ok

File: source/_components/egardia.markdown (hunk 3 above)
Comment (jeroenterheerdt) [reply source/_components/egardia.markdown]: ok

File: source/_components/egardia.markdown (hunk 4 above)
Comment (jeroenterheerdt) [reply source/_components/egardia.markdown]: ok.

File: source/_components/egardia.markdown (hunk 5 above)
Comment (MartinHjelmare) [reply source/_components/egardia.markdown]: Ok, then this needs to be clarified. Eg, for how long does the user need to have the script running. I'm guessing the user needs to actively shut the server down, by exiting the script.

Comment (jeroenterheerdt) [reply source/_components/egardia.markdown]: Fair enough, I tried to make it more clear. Hope this helps.

File: source/_components/egardia.markdown [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+```yaml
+# Example configuration.yaml entry
+egardia:
+  host: YOUR_HOST
+  username: YOUR_USERNAME
+  password: YOUR_PASSWORD
+```
+
+Configuration variables:
```
Comment (MartinHjelmare) [inline source/_components/egardia.markdown]: Please update this to use the configuration tag:
https://home-assistant.io/developers/documentation/create_page/#configuration

Comment (balloob) [reply source/_components/egardia.markdown]: I want to do the release right now so going ahead and merge this. @jeroenterheerdt please address this comment in a new PR.

Comment (jeroenterheerdt) [reply source/_components/egardia.markdown]: Sure, will do.

**Pull request merged**